content/docs/user-guide/components/reference/vegetation/vegetation-area-system.md

```diff
@@ -0,0 +1,73 @@
+
+## Provider
+
+[Vegetation Gem](/docs/user-guide/gems/reference/environment/vegetation/)
+
+## Source code
+
+[`/Gems/Vegetation/Code/Source/AreaSystemComponent.h`](https://github.com/o3de/o3de/blob/development/Gems/Vegetation/Code/Source/AreaSystemComponent.h)
+
+## Parameters
+
+| Property | Description | Values | Default |
+| --- | --- | --- | --- |
+| **View Area Grid Size** | The number of sectors (per-side) of a managed grid in a scrolling view centered around the camera | 1 to 128 | 13  |
+| **Sector Point Density** | The number of equally spaced vegetation instance grid placement points (per-side) within a sector | 1 to 64 | 20 |
+| **Sector Size In Meters** | The size in meters per-side of each sector. | 1 to 1024 | 16 |
+| **Thread Processing Interval** | The delay in milliseconds between processing queued thread tasks. | 0 to 5000 | 500 |
+| **Sector Search Padding** | Increases the search radius for surrounding sectors when enumerating through vegetation instances. | 0 to 2 | 0 |
+| **Sector Point Snap Mode** | Controls whether vegetation placement points are located at the corner or the center of the cell. | `Corner`, `Center` | `Center` |
+
+## AreaSystemRequestBus
+
+File: [`Gem/sVegetation/Code/Include/Vegetation/Ebuses/AreaSystemRequestBus.h`](https://github.com/o3de/o3de/blob/5783c18cc8d65f75737159f4cdcf6019d0e8dcfc/Gems/Vegetation/Code/Include/Vegetation/Ebuses/AreaSystemRequestBus.h)
+
+| Request Name | Description | Parameters | Return | Scriptable |
+| --- | --- | --- | --- | --- |
+| `RegisterArea` | Adds a Vegetation Area entity to the system for processing. | `AZ::EntityId`, `AZ::u32`, `AZ::Aabb&` | None | No  |
+| `UnregisterArea` | Removes a Vegetation Area entity from the system. | `AZ::EntityId` | None | No  |
+| `RefreshArea` | Queues a Vegetation Area to update its data within the system and mark regions of sectors dirty for reprocessing. | `AZ::EntityId`, `AZ::u32`, `AZ::u32`, `AZ::Aabb&` | None | No  |
+| `RefreshAllAreas` | Updates the cached data for all registered areas and causes the entire system to repopulate. | None | None | No  |
+| `ClearAllAreas` | Forces all cached data and instances in every sector to be removed, and causes the entire system to repopulate. | None | None | No  |
+| `MuteArea` | Tells the Vegetation Area System to ignore the specified area when refilling sectors because it's managed elsewhere. This is the case with areas controlled by Vegetation Layer Blenders. | `AZ::EntityId` | None | No  |
+| `UnmuteArea` | Re-enables direct processing of a Vegetation Area that was previously muted. | `AZ::EntityId` | None | No  |
+| `EnumerateInstancesInOverlappingSectors` | Visit all instances contained within every vegetation sector that overlaps the given bounds, until the callback function decides otherwise. Additionally, the sector boundary is by the **Sector Search Padding** value in the Area System component's configuration. | `AZ::Aabb&`, `AreaSystemEnumerateCallback` | None |     |
```

Comment:
```suggestion
| `EnumerateInstancesInOverlappingSectors` | Visit all instances contained within every vegetation sector that overlaps the given bounds, until the callback function decides otherwise. Additionally, the sector boundary is by the **Sector Search Padding** value in the Area System component's configuration. | `AZ::Aabb&`, `AreaSystemEnumerateCallback` | None |  No  |
```